index.html

```
@@ -1216,14 +1147,14 @@
         <span class="text-primary italic">Org Finder</span>
       </span>
       </a>
+      <nav class="hidden md:flex items-center gap-6 text-sm font-medium">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </nav> ], start tag match failed [ <nav class="hidden md:flex items-center gap-6 text-sm font-medium"> ] on line 1150.

(tag-pair)